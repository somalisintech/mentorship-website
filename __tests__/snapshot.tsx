import { render } from '@testing-library/react';
import Home from '@/pages/index';
import About from '@/pages/about';
import Profile from '@/pages/profile';
import Mentors from '@/pages/mentors';

it('<Home /> should render "Landing page"', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});

it('<About /> should render "About page"', () => {
  const { container } = render(<About />);
  expect(container).toMatchSnapshot();
});

it('<Mentors /> should render "Mentors page"', () => {
  const { container } = render(<Mentors />);
  expect(container).toMatchSnapshot();
});

it('<Profile /> should render "User profile page"', () => {
  const { container } = render(<Profile />);
  expect(container).toMatchSnapshot();
});
