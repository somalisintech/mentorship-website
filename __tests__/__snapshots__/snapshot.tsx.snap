// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render "About page" 1`] = `
<div>
  <h1
    class="my-auto text-4xl text-center"
  >
    About page
  </h1>
</div>
`;

exports[`<Home /> should render "Landing page" 1`] = `
<div>
  <h1
    class="my-auto text-4xl text-center"
  >
    Landing page
  </h1>
</div>
`;

exports[`<Mentors /> should render "Mentors page" 1`] = `
<div>
  <h1
    class="my-auto text-4xl text-center"
  >
    Mentors page
  </h1>
</div>
`;

exports[`<Profile /> should render "User profile page" 1`] = `
<div>
  <h1
    class="my-auto text-4xl text-center"
  >
    User profile page
  </h1>
</div>
`;
